### DISCUSSION TITLE
Windows 11 WSL2 Docker Setup with Cuda, I chose to run via 'option 3' --xformers woes

### DISCUSSION THREAD

**Discussion by nbollman** [Josh-XT/AGiXT#862] [Q&A] [answered]
Im running text-generation-webui locally already with xformers, but when I chose to modify the command line argumetns installing through XGiXT for launching text-generation-webui it did so without actually trying to install xformers as it does in the oobabooga setup... Im not very good with docker, and I can see the CLI_ARGS but dont see an option for changing them, and I cant launch the containter beccause of the failed argument. Id like it to run with xformers to increase the token count, could someone kindly push me in the right direction, or even unkindly... only the right direction is necessary. :)

**Comment by mongolu** [comment, marked as answer]
Option 3 means Nvidia with Docker, right?
If so and CLI_ARGS are bad defined initially, you could edit .env file present in the AGiXT root folder.

**Reply by nbollman** [reply]
correct on option 3, nvidia with docker, Ive managed to get everything to launch, text-generation-webui (without xformers for now) ... now its postgress holding me back... Ive never been a huge fan of docker... I like anaconda environments...

**Comment by nbollman** [comment]
and editing the .env file was a good answer thanks
